File: JS/51-60/e_52.js
function concatListas(L1, L2, L3) {
    return L1.concat(L2, L3);
}

// Ejemplo de uso:
let L1 = [1, 2, 3];
let L2 = [4, 5, 6];
let L3 = [7, 8, 9];

let resultado = concatListas(L1, L2, L3);
console.log(resultado); // Devuelve [1, 2, 3, 4, 5, 6, 7, 8, 9]


// Esta función utiliza el método concat() de las listas para concatenar las tres listas y devuelve la lista resultante.

File: Python/51-60/e_52.py
# Escribe una función en Python concatListas(L1,L2,L3) que tome tres listas L1, L2 y L3 como parámetros,
# y devuelva una concatenación de estas tres listas.


def concatListas(L1, L2, L3):
    return L1 + L2 + L3

# Ejemplo de uso:
L1 = [1, 2, 3]
L2 = [4, 5, 6]
L3 = [7, 8, 9]

resultado = concatListas(L1, L2, L3)
print(resultado)  # Devuelve [1, 2, 3, 4, 5, 6, 7, 8, 9]


# Esta función simplemente utiliza el operador + para concatenar las tres listas y devuelve la lista resultante.